prod/westeurope/internal/api/apim/api_management_api_cgn_merchant/terragrunt.hcl

```diff
@@ -0,0 +1,32 @@
+dependency "api_management" {
+  config_path = "../api_management"
+}
+
+# Internal
+dependency "resource_group" {
+  config_path = "../../../resource_group"
+}
+
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_api_management_api?ref=v2.1.35"
+}
+
+inputs = {
+  name                  = "io-cgn-merchant-api"
+  resource_group_name   = dependency.resource_group.outputs.resource_name
+  api_management_name   = dependency.api_management.outputs.name
+  revision              = "1"
+  display_name          = "IO CGN MERCHANT API"
+  description           = "CGN MERCHANT API for IO platform."
+  host                  = "api.io.italia.it"
+  path                  = "cgnmerchant"
```

Comment:
what does this path means @uolter? I'd rather assume that API is called with `https://api.io.italia.it/api/v1/merchant/cgn/{restOfPath}` isn't it?

Reply:
ok if public API is `https://api.io.italia.it/api/v1/merchant/cgn/{restOfPath}` we need to set as follow

api_management_api_cgn_merchant/terragrunt.hcl
`path = api/v1/merchant/cgn`

api_management_api_cgn_merchant/policy.xml
`<set-backend-service base-url="{{io-fn-cgnmerchant-url}}/api/v1/merchant/cgn" />`